troncos/traces/http.py

```diff
@@ -46,6 +46,7 @@ def create_http_span(
     """
 
     attr = {
+        "resource": f"{http_req_method} {http_req_url}",
```

Comment:
I want this to be the "template path" that handles the view and not just the URL.

Example:

`GET /api/v1/users/{userid}` instead of `GET http://print-service.logistics/api/v1/users/12930`. 

That lets us see all requests to one "view" in the app as one resource.

Reply:
Looking at the information we get in the middleware, we can easily `/api/v1/users/12930`, having a hard time seeing how we can provide `/api/v1/users/{userid}`. Any ideas?

Reply:
I see, this involves ditching the middleware and patching the fastapi decorators. I can look into that :+1:

Reply:
They also use the Generic ASGIMiddlware to do distributed tracing.

Reply:
Take a look now. Does this make sense?

![image](https://user-images.githubusercontent.com/6891726/195361734-7fbdf6c9-2b3d-4825-af19-9d7fd54f6a71.png)